packages/ember-toucan-form/package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "@crowdstrike/ember-toucan-form",
+  "version": "0.0.0",
+  "description": "Build forms with ember-headless-form and ember-toucan-core",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:CrowdStrike/ember-toucan-core.git",
+    "directory": "packages/ember-toucan-form"
+  },
+  "license": "Apache-2.0",
+  "author": "CrowdStrike UX Team",
+  "files": [
+    "addon-main.cjs",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js:fix": "eslint . --fix",
+    "lint:types": "glint",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
+    "prepack": "rollup --config",
+    "format:check": "prettier -c .",
+    "format": "prettier -w ."
+  },
+  "peerDependencies": {
+    "@crowdstrike/ember-toucan-styles": "^2.0.1",
+    "@ember/test-helpers": "^2.8.1",
+    "@glimmer/tracking": "^1.1.2",
+    "autoprefixer": "^10.0.2",
+    "ember-source": ">=4.8.0",
+    "fractal-page-object": "^0.3.0",
+    "postcss": "^8.2.14",
+    "tailwindcss": "^2.2.15 || ^3.0.0"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.20.7",
+    "@embroider/addon-shim": "^1.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.0",
+    "@babel/eslint-parser": "^7.19.1",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-proposal-decorators": "^7.17.0",
+    "@babel/plugin-syntax-decorators": "^7.17.0",
+    "@babel/preset-typescript": "^7.18.6",
+    "@crowdstrike/ember-toucan-styles": "^2.0.1",
+    "@ember/test-helpers": "^2.8.1",
+    "@embroider/addon-dev": "^3.0.0",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "@glint/core": "^1.0.0-beta.3",
+    "@glint/environment-ember-loose": "^1.0.0-beta.3",
+    "@glint/template": "^1.0.0-beta.3",
+    "@nullvoxpopuli/eslint-configs": "^3.0.4",
+    "@tsconfig/ember": "^2.0.0",
+    "@types/ember": "^4.0.0",
+    "@types/ember__application": "^4.0.0",
+    "@types/ember__array": "^4.0.0",
+    "@types/ember__component": "^4.0.0",
+    "@types/ember__controller": "^4.0.0",
+    "@types/ember__debug": "^4.0.0",
+    "@types/ember__engine": "^4.0.0",
+    "@types/ember__error": "^4.0.0",
+    "@types/ember__object": "^4.0.0",
+    "@types/ember__polyfills": "^4.0.0",
+    "@types/ember__routing": "^4.0.0",
+    "@types/ember__runloop": "^4.0.0",
+    "@types/ember__service": "^4.0.0",
+    "@types/ember__string": "^3.16.0",
+    "@types/ember__template": "^4.0.0",
+    "@types/ember__test": "^4.0.0",
+    "@types/ember__utils": "^4.0.0",
```

Comment:
Looking at some of these, I can probably slim this out a bit.  I'll do that in a follow up PR as this `package.json` matches what's in `packages/ember-toucan-core`